pax-compiler/templates/cartridge-lib.tera

```diff
@@ -81,7 +81,7 @@ pub fn instantiate_expression_table<R: 'static + RenderContext>() -> HashMap<usi
         {% endif %}
 
         TypesCoproduct::{{ expression_spec.pascalized_return_type }}(
-            {{ expression_spec.output_statement }}
+            {{ expression_spec.output_statement }}.into()
```

Comment:
I believe this is problematic.  I set up repro here:
https://github.com/pax-lang/pax/compare/6df4802c6b32%5E...78fb7765dcb2#diff-22275c553b4a7bfe13d35b04f59015dba9373818c30049484349acdb2a4627f4L3

I started with your branch, rebased off of `origin/master` so we have Warfa's latest work on `/examples`, then modified  the `fireworks` example to be able to reproduce the error I'm seeing related to this `.into`.

You should be able to reproduce by pulling the branch `oa/pax_qualia` (from this repo) and running `cargo run --examples fireworks`.  The error:

```
error[E0277]: the trait bound `isize: From<i32>` is not satisfied
   --> /Users/zack/code/pax/examples/src/fireworks/.pax/pkg/pax-cartridge/src/lib.rs:110:19
    |
110 |             0..60.into()
    |                   ^^^^ the trait `From<i32>` is not implemented for `isize`
```

This is on the range expression `for i in 0..60` — the addition of the `.into` breaks that range literal.  If we need to add an additional `.into` for Color, can it be higher in the codegen than this terminal statement?   Because of the constraints around the graph of types we can `into`, each new `into` comes with a cost, so is it possible to drop this one entirely?